docs/specifying-a-call-to-configure.md

```diff
@@ -51,6 +51,17 @@ A.CallTo(() => deepThought.Invoke("What is the answer to life, the universe, and
 A.CallTo(() => deepThought("What is the answer to life, the universe, and everything?")).Returns(42);
 ```
 
+## Specifying a call to an explicitly implemented interface member
+
+An explicitly implemented member is not directly visible on the concrete, it has to be called via the interface.
+In addition, since it's not possible to just override an explicitly implemented member in a derived class, you
+need to explicitly specify that the fake implements the interface:

```

Comment:
```suggestion
An explicitly implemented member is not directly visible on the concrete class. Instead, it has to be called
(or overridden) via the interface. In addition, since fakes don't automatically intercept explicitly implemented
interfaces, you need to explicitly specify that the fake implements the interface:
```

Reply:
Perfect, thanks for the suggestion!

Reply:
I applied your suggestion but now I need to squash...